src/strands/agent/agent.py

```diff
@@ -597,6 +624,34 @@ async def _execute_event_loop_cycle(self, invocation_state: dict[str, Any]) -> A
             async for event in events:
                 yield event
 
+    def _convert_prompt_to_messages(self, prompt: str | list[ContentBlock] | Messages | None) -> Messages:
```

Comment:
I'm really glad we didn't add a `Message` parameter to `StartInvocationEvent` - it should now be `messages` :)